src/main/resources/camel/obs/obs-to-odoo-resource.xml

```diff
@@ -120,7 +120,7 @@
                         </choice>
                     </when>
                     <otherwise>
-                        <log loggingLevel="DEBUG" message="Skipping Obs because the question concept doesn't match any configured question" />
+                        <log loggingLevel="DEBUG" message="Skipping Obs because the question concept doesn't match any configured question or is null" />
```

Comment:
This change will break some existing tests @icrc-fdeniger. Those will need to be aligned.